doc/source/reference/arrays.classes.rst

```diff
@@ -39,7 +39,7 @@ Special attributes and methods
 
 Numpy provides several hooks that classes can customize:
 
-.. function:: class.__numpy_ufunc__(self, ufunc, method, i, inputs, **kwargs)
+.. method:: class.__numpy_ufunc__(ufunc, method, i, inputs, **kwargs)
 
    .. versionadded:: 1.10
```

Comment:
Happened to notice this: actually not added in 1.10 in the end -- should become 1.11.